packages/fxa-content-server/app/scripts/templates/settings/totp_secret.mustache

```diff
@@ -0,0 +1,13 @@
+<div id="recovery-codes">
```

Comment:
nit: Since we are adding more modals for 2FA I wonder if we should group them in a folder?

Reply:
Yeah, this is lazy of me, naming it the same to reuse the styles...

Reply:
I just tried to refactor the css but hit a semi-major issue with selector specificity. Not a quick fix so I'm going to defer it